ouroboros-consensus/src/Ouroboros/Consensus/Network/NodeToNode.hs

```diff
@@ -241,6 +242,12 @@ defaultCodecs ccfg version = Codecs {
     dec :: SerialiseNodeToNode blk a => forall s. Decoder s a
     dec = decodeNodeToNode ccfg version
 
+    encHash :: HeaderHash blk -> Encoding
+    encHash = Serialise.encode . toRawHash (Proxy @blk)
+
+    decHash :: forall s. Decoder s (HeaderHash blk)
+    decHash = fromRawHash (Proxy @blk) <$> Serialise.decode
```

Comment:
It's the third time I see these now, we should define them near `ConvertRawHash`

Reply:
Ah, okay, will do.